Monika After Story/game/script-topics.rpy

```diff
@@ -3641,13 +3641,19 @@ label monika_trolley:
     m 1eua "Oh, cool. I love thinking about these sorts of thought experiments."
     m 1euc "I guess we're talking about real people, right? I wouldn't have a particular preference if I knew they didn't have free will."
     m 1dsc "Hmmm..."
-    m 1eua "If we were talking about the classic problem, of course I would switch."
-    m 1rkc "But if we're talking about real people..."
-    m 3esc "I wouldn't switch."
-    m 4esc "That's the only logical choice, if you think about it."
-    m 1euc "It feels nice to say that you'd save the most people, doesn't it? But people only say that because nobody in the example matters to them."
-    m "If they actually cared about the person, they'd kill the other five in a heartbeat."
-    m 2tsb "Do you know anyone who would want a self-driving car to save other people by killing the driver? Didn't think so, [player]."
+    m 3esa "The classic trolley problem has us choose between letting the trolley run over five people, or switching to the track where only one person will be killed."
+    m 1esa "The problem is mostly famous due to how divisive it is."
+    m "Regardless of whether they'd pull the switch or not, most people believe that their choice simply must be the correct one."
```

Comment:
I think this goes without saying for most famous questions, so I don't think we need to say this.

Reply:
That line is meant more as a lead-in for the next one than as a standalone. I feel like skipping over it would leave a discontinuity and I wasn't sure what else to put there.
But addressing your point here, you also have problems which, while having a single correct solution, are famous for being absurdly hard (this is more of a math thing than philosophy), or problems which are famous for straight up not having a solution, since trying to find one leads you to a logical contradiction. Things like Liar's or Russell's paradox.